src/Bundle/DependencyInjection/DoctrineBehaviorsExtension.php

```diff
@@ -4,29 +4,44 @@
 
 namespace Knp\DoctrineBehaviors\Bundle\DependencyInjection;
 
-use Doctrine\Common\EventSubscriber;
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Extension\Extension;
-use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
+use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
 
 final class DoctrineBehaviorsExtension extends Extension
 {
-    /**
-     * @var string
-     */
-    private const DOCTRINE_EVENT_SUBSCRIBER_TAG = 'doctrine.event_subscriber';
-
     /**
      * @param string[] $configs
      */
     public function load(array $configs, ContainerBuilder $containerBuilder): void
     {
-        $loader = new YamlFileLoader($containerBuilder, new FileLocator(__DIR__ . '/../../../config'));
-        $loader->load('services.yaml');
+        $loader = new XmlFileLoader($containerBuilder, new FileLocator(dirname(__DIR__) . '/../Resources/config'));
+        $loader->load('services.xml');
```

Comment:
Keep services in `services.yaml`. That's unrlated change and I want to stick with YAML

Reply:
Ok. I'll do this. Just to understand : why ? ^^

Reply:
Yaml is more friendly to new devs, as it's default in Symfony docs and StackOverflow answers.

Reply:
Ok for application and config override but for internal bundle config, XML is the standard, no?

Reply:
Done